### ISSUE TITLE
evaluation performance on Kitti

### ISSUE THREAD

**Issue by gongshichina** [xingyul/flownet3d#22]
Thanks for your open code.
When I use your pre-trained model and processed kitti dataset for a evaluation, with setting rgb zeros, valid mask ones, the evaluation performance is much lower than which reported in your paper.

eval mean loss: 0.024921
eval mean EPE 3D: 0.170297    (0.122)
eval mean ACC 3D: 0.101357   (0.0561)
eval mean ACC 3D 2: 0.372272

**Comment by xingyul** [comment]
The results in Table 4 of the paper is on KITTI point clouds after ground removal.

**Comment by s-ryosky** [comment]
Hi @xingyul, 
In your paper, how many points did you use on KITTI evaluation?

**Comment by xingyul** [comment]
All of the points.

**Comment by haoyu6427** [comment]
> Thanks for your open code.
> When I use your pre-trained model and processed kitti dataset for a evaluation, with setting rgb zeros, valid mask ones, the evaluation performance is much lower than which reported in your paper.
> 
> eval mean loss: 0.024921
> eval mean EPE 3D: 0.170297 (0.122)
> eval mean ACC 3D: 0.101357 (0.0561)
> eval mean ACC 3D 2: 0.372272

So the processed kitti dataset for evaluation has the ground points and the corresponding performance paper reported is 0.211? But the readme claims that KITTI point clouds have removed the ground points.
